lib/api/apiUtils/object/locationStorageCheck.js

```diff
@@ -22,7 +22,7 @@ function _gbToBytes(gb) {
 function locationStorageCheck(location, updateSize, log, cb) {
     const lc = config.locationConstraints;
     const sizeLimitGB = lc[location] ? lc[location].sizeLimitGB : undefined;
-    if (updateSize === 0 || sizeLimitGB === undefined) {
```

Comment:
Are you sure this fixes the issue? Cos when I tested, if `lc[location].sizeLimitGB` was set to null, `sizeLimitGB` was set to undefined

Reply:
maybe different node versions?, I'm definitely able to reproduce on my end.

Reply:
Scratch that, I realize what I did wrong. Thx for fixing 👍